modules/ROOT/pages/functions/faq.adoc

```diff
@@ -0,0 +1,258 @@
+Serverless Functions FAQ
+========================
+
+Serverless Functions frequently asked questions.
+
+
+== What are Serverless Functions?
+
+Serverless Functions are a collection of tools that enable developers to create
+and run functions as a Knative Service on Kubernetes. The major components of
+this project are listed here.
+
+* Function runtimes for Go, Node.js and Quarkus
+* Buildpacks for Go, Node.js and Quarkus
+* A CLI for creating and managing functions as a Knative Service
+* Knative Serving and Eventing as the platform on which it all runs
+
+== How mature is the Serverless Functions Project?
+
+OpenShift Serverless Functions are currently in Developer Preview.
+
+== How does this project relate to Knative?
+
+Knative is the target deployment platform for Serverless Functions.
```

Comment:
Again, this should be a given since it's in your prereqs but up to you whether to include.

Tbh I don't know if we'll include this FAQ in OCP docs anyway, the information that's useful from it will probably be moved into some proper "about functions" guide or something. We don't tend to have things like FAQs in downstream docs afaik.